assets/js/components/events/EventsDashboard.jsx

```diff
@@ -32,19 +32,25 @@ const base64ToHex = str => {
   return result.toUpperCase();
 }
 
-const categoryTag = (category, subCategories) => {
+const integrationErrorTag = () => (
+  <Tooltip title="Integration Response Error">
+    <ShrinkOutlined style={{color: 'red', fontSize: 20}} />
+  </Tooltip>
+);
+
+const categoryTag = (category, subCategories, integrationError) => {
   switch(category) {
     case "uplink_dropped":
       return <Text>Uplink Dropped</Text>;
     case "uplink":
-      return <Text>Uplink</Text>;
+      return <Text>Uplink {integrationError ? integrationErrorTag() : ''}</Text>;
```

Comment:
can probably just be

integrationError && integrationErrorTag()

?

Reply:
I really need to tone down my obsession w/ ternaries. Fixed!